frontend/src/global.js

```diff
@@ -18,6 +18,23 @@ export const navlinks = [
     },
 ];
 
+export const stockData = [
+    {
+        "c": 261.74,
+        "High Price": 263.31,
+        "Low Price": 260.68,
+        "Open Price": 261.07,
+        "t": 1582641000 
+    },
+    {
+        "c": 261.74,
+        "High Price": 263.31,
+        "Low Price": 260.68,
+        "Open Price": 261.07,
+        "t": 1582641000 
+    },
```

Comment:
just wondering why do you have this object twice? I rendered it, and it looks nice! The card with the stock info showed up twice, so I'm wondering if that was intentional

Reply:
It's intentional. Right now, I am assuming the stock info will be a list of quotes. That is why I have the object twice.